rfc8999.md

```diff
@@ -9,6 +9,7 @@ consensus: true
 ipr: trust200902
 area: Transport
 workgroup: QUIC
+keyword: [next generation, protocol, transport, secure, crypto]
```

Comment:
+ UDP

Reply:
That's in the title, so I'm OK with leaving it out.  Besides, it's not that important.

Reply:
It's a redundancy symbol incase the first one gets lost. But I see that fec is mentioned in recovery. I will not die here.